h2o-hadoop/scripts/docker/common/startup.sh

```diff
@@ -0,0 +1,70 @@
+#! /bin/bash
```

Comment:
Can we use the existing scripts to execute the tests?

See scripts/run.py. Option '--onHadoop' lets you run H2O on Hadoop. It would be great if we could re-use the test framework as is.